web/templates/settings/invites_list_admin.hbs

```diff
@@ -5,7 +5,7 @@
     {{/unless}}
     <a class="invite-user-link" role="button"><i class="fa fa-user-plus" aria-hidden="true"></i>{{t "Invite users to organization" }}</a>
 
-    <div class="settings_panel_list_header">
+    <div id="invite-filter-input" class="settings_panel_list_header">
```

Comment:
Can you name it something else since this div is also containing things other than the input.